irr/src/OpenGL/ExtensionHandler.h

```diff
@@ -165,13 +165,16 @@ class COpenGL3ExtensionHandler : public COGLESCoreExtensionHandler
 
 	inline void irrGlObjectLabel(GLenum identifier, GLuint name, const char *label)
 	{
+		// KHR_debug implements ObjectLabelKHR in OpenGL ES
+#ifndef _IRR_COMPILE_WITH_OGLES2_
```

Comment:
IIRC this was due to a bug in our code, the comment should be clearer why this is excluded

also: ifdef is compile time, but I can still choose gl3 at runtime -> will exclude it in more cases than needed